cowprotocol/cowswap
# [EthFlow]#1298 wire up cancellation

# Summary

Closes #1297 and #1298
Waterfalls onto https://github.com/cowprotocol/cowswap/pull/1628
Which waterfalls onto https://github.com/cowprotocol/cowswap/pull/1613
Which waterfalls onto https://github.com/cowprotocol/cowswap/pull/1546

Final *working* part of the Native EthFlow cancellation series.
Please review the code in the PRs listed above

# To Test

1. Turn eth-flow on with `localStorage.setItem('enableEthFlow', '1')`
2. Place regular order
3. Cancel it on Activity modal
* Should work as usual
4. Place limit order
5. Cancel it on limit orders UI
* Should work as usual
6. Place ethflow order
7. Cancel it on Activity modal
* Should trigger an on-chain tx
* Order should be cancelled as usual

# Notes
- State not yet synced with ethflow stepper
- I expect many small issues, text adjustments and etc which I'll address in follow up PRs. Please let me know what's missing anyway
